src/iuse.cpp

```diff
@@ -4156,7 +4156,7 @@ int iuse::radio_on(player *p, item *it, bool t, point pos)
             }
 
             std::vector<std::string> segments = foldstring(message, RADIO_PER_TURN);
-            int index = calendar::turn % (segments.size());
+            int index = calendar::is_time_for(segments.size());
```

Comment:
That is not the same, `is_time_for` returns a boolean, but the index is required here. This to make the radio print the next line of the radio message. Using a boolean would mean the index is always either 0 or 1 (which btw. in the case of a message with only one line leads to a crash).